xinference/model/llm/llm_family.json

```diff
@@ -1522,5 +1702,31 @@
         "model_revision": "fa88072fee36e36282287410e00897df2f59e09b"
       }
     ]
+  },
+  {
+    "version": 1,
+    "context_length": 2048,
+    "model_name": "TinyLlama",
```

Comment:
All of our model names are in lowercase. To follow the convention, how about using the name `tiny-llama`?